app/style/SecurityCenterMessagePage.less

```diff
@@ -10,7 +10,7 @@
 .security-center-form {
   background-color: #fff;
   padding: 1em;
-  padding-top: 40px;
+  padding: 40px;
```

Comment:
Drop the first padding as it is redundant

Reply:
oops sorry, missed that lol